packages/frontend/components/ui/checkbox.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+
+function Checkbox({
+  name,
+  checked,
+  type,
+  disabled,
+  className,
+  value,
+  label,
+  required,
+  onChange,
+  ...otherProps
+}) {
+  function changeValue(targetValue) {
+    if (onChange) {
+      onChange(targetValue);
+    }
+  }
+
+  let customStyleName = '';
+  if (checked && type === 'success') {
+    customStyleName = 'success-check-box';
+  } else if (checked && type === 'danger') {
+    customStyleName = 'danger-check-box';
+  }
+
+  return (
+    <div className="flex items-center">
+      <input
+        id={name}
+        name={name}
+        type="checkbox"
+        checked={checked}
+        className={cls(`default-check-box ${customStyleName}`, className)}
+        disabled={disabled}
+        onChange={e => changeValue(e.currentTarget.checked)}
+        required={required}
+        {...otherProps}
+      />
+      <label
```

Comment:
what you think about adding conditional rendering like `{label && <label>...</label>}` in here? i didn't think that can we use checkboxes without label but having a feature like that can be good.

Reply:
Conditional rendering would be good. I agree.

Reply:
I just realized now, When the label is not rendered, css needs to changed totally. Because of css structure to change default mark, I created custom css based on 'input field + label' 
For now I think we can leave it as is.